include/plat/marvell/a8k/common/plat_config.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2018 Marvell International Ltd.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ * https://spdx.org/licenses
+ */
+
+#ifndef __BOARD_CONFIG_H__
```

Comment:
Please modify guard macros to reflect file name. Also fix other instances of this problem.